frontend/src/features/mapSideBar/SidebarContents/subforms/ClassificationForm.tsx

```diff
@@ -0,0 +1,126 @@
+ */
+
+const Title = styled.h4`
+  float: left;
+`;
+
+/** formated information box to display the classification definitions to the right of the select */
+const InfoBox = styled.div`
+  border: 1px solid #f2f2f2;
+  border-radius: 4px;
+  text-align: left;
+  padding: 8px 12px 10px;
+`;
+
+/** formatted field description that will appear underneath the select input */
+const FieldDescription = styled.p`
+  margin-left: 152px;
+  font-size: 12px;
+  width: 250px;
+  text-align: left;
+`;
+
+/** used to conditionally render the information display box with the desired content */
+const InfoBoxWithContent = (content: string) => {
+  return (
+    <InfoBox>
+      <p style={{ fontSize: '11px' }}>{content}</p>
+    </InfoBox>
+  );
+};
+
+interface IClassificationFormProps {
+  /** title to be displayed at the top of the form */
+  title?: string;
+  /** the classification set to be used with the component, could vary by form */
+  classifications: SelectOptions;
+  /** the field label for the select options (eg) Building Classification/Land Classification */
+  fieldLabel?: string;
+  /** pass down the parents nameSpace in order to access the appropriate fields */
+  nameSpace?: string;
+  /** pass the index to access the appropriate property if it is in a list format */
+  index?: string;
+  /** pass a tooltip for the classification field */
+  toolTip?: string;
+}
+
+export const ClassificationForm: React.FC<IClassificationFormProps> = ({
+  title,
+  classifications,
+  fieldLabel,
+  nameSpace,
+  index,
+  toolTip,
+}) => {
+  const formikProps = useFormikContext();
+
+  /** classId based on current formik values to determine which classsification information box to display */
+  let classId = index
+    ? (formikProps.values as any).data.buildings[index].classificationId
+    : (formikProps.values as any).data.classificationId;
+
+  const withNameSpace: Function = React.useCallback(
+    (name?: string) => {
+      return [nameSpace ?? '', name].filter(x => x).join('.');
+    },
+    [nameSpace],
+  );
+
+  const renderInfo = () => {
+    switch (classId) {
+      case CLASSIFICATIONS.CoreOperational:
+        return InfoBoxWithContent('Core Operational');
+      case CLASSIFICATIONS.CoreStrategic:
+        return InfoBoxWithContent('Core Strategic');
+      case CLASSIFICATIONS.SurplusActive:
+        return InfoBoxWithContent('Surplus Actice');
+      case CLASSIFICATIONS.SurplusEncumbered:
+        return InfoBoxWithContent('Surplus Encumbered');
+    }
+  };
+
+  return (
+    <>
+      <Row>
+        <Title>{title}</Title>
+      </Row>
+      <Row>
+        <Col md={6}>
+          <Row style={{ display: 'flex' }}>
+            <Label required>{fieldLabel}</Label>
+            <FastSelect
+              formikProps={formikProps}
+              type="number"
+              style={{ marginTop: '5px', display: 'flex' }}
+              placeholder="Must Select One"
+              field={withNameSpace('')}
```

Comment:
no field name? or is this coming later?

Reply:
Woops I think I can just pass it nameSpace as it is defined in the parent